fastlane/lib/fastlane/lane_manager.rb

```diff
@@ -104,17 +104,31 @@ def self.print_table(actions)
     # @param platform: is probably nil, but user might have called `fastlane android`, and only wants to list those actions
     def self.choose_lane(ff, platform)
       loop do
-        UI.error "You must provide a lane to drive. Available lanes:"
-        available = ff.runner.available_lanes(platform)
-
+        UI.message "Welcome to fastlane! Here's what your app is setup to do:".white
+        available = ff.runner.lanes[platform].to_a
+        rows = []
         available.each_with_index do |lane, index|
-          UI.message "#{index + 1}) #{lane}"
+          row = [index + 1, lane.last.pretty_name, lane.last.description.first]
+          rows << row
+        end
+
+        if rows.size > 0
+          rows << [rows.size + 1, "cancel", "No selection!"]
         end
 
+        table = Terminal::Table.new(
+          title: "Available lanes to run".white,
+          headings: ['Number'.white, 'Lane Name'.white, 'Description'.white],
+          rows: rows
+        )
+        puts table
+
+        UI.message "Which number would you like run?".white
+
         i = $stdin.gets.strip.to_i - 1
         if i >= 0 and available[i]
-          selection = available[i]
-          UI.important "Driving the lane #{selection}. Next time launch fastlane using `fastlane #{selection}`"
+          selection = available[i].last.pretty_name
+          UI.important "Great, we'll run the lane `#{selection}`. You can do this directly typing `fastlane #{selection}`."
```

Comment:
Who is `we`? We haven't used that wording anywhere else before I think. How about just 

```
Running lane `#{selection}`. Next time you can do this directly by typing `fastlane #{selection}`.
```